meerk40t/core/cutplan.py

```diff
@@ -478,15 +496,19 @@ def clear(self):
         self.commands.clear()
 
 
-def is_inside(inner, outer):
+def is_inside(inner, outer, tolerance=0):
```

Comment:
So if we have two overlapping circles are they both inside each other? Then which goes first? I guess it doesn't matter since they are overlapping.

Reply:
The tpology is a bit hard to imagine where that would be the case, in essence if the diameter of the circle would be smaller than the given tolerance then that woul happen. Will try it out what happens...